src/components/edit-asset-address-form/editable-address/editable-address.tsx

```diff
@@ -194,7 +262,14 @@ export const EditableAddress = ({
                     : "govuk-form-group lbh-form-group"
                 }
               >
-                <label className="govuk-label lbh-label" htmlFor="postcode">
+                <label
+                  className={
+                    addressEditSuccessful
+                      ? "govuk-label lbh-label grey-text"
+                      : "govuk-label lbh-label"
+                  }
+                  htmlFor="postcode"
+                >
```

Comment:
[This library](https://yarnpkg.com/package/classnames) is perfect for this usecase.